diff --git a/scripts/server.cjs b/scripts/server.cjs
index 84e9e6d5..8fb2832a 100644
--- a/scripts/server.cjs
+++ b/scripts/server.cjs
@@ -18,6 +18,10 @@ const handler = (req, resp) => {
 	const fallback = () => {
 		fs.readFile(path.join(__dirname, '..', 'build', htmlFile), (err, buf) => {
 			resp.setHeader('Content-Type', 'text/html');
+			resp.setHeader('Access-Control-Allow-Origin', '*');
+			resp.setHeader('Access-Control-Allow-Methods', 'GET, POST, OPTIONS, PUT, PATCH, DELETE');
+			resp.setHeader('Access-Control-Allow-Headers', 'X-Requested-With,content-type');
+			resp.setHeader('Access-Control-Allow-Credentials', true);
 			resp.end(buf);
 		});
 	};
@@ -50,4 +54,4 @@ if(useHTTPS) {
 	http.createServer(handler).listen(port, () => {
 		console.log(`>>> Listening on http://0.0.0.0:${port}/\n`);
 	});
-}
\ No newline at end of file
+}
